all/build.gradle

```diff
@@ -4,7 +4,10 @@ description = "gRPC: All"
 
 buildscript {
     repositories {
-        mavenCentral()
+        maven {
+            // The google mirror is less flaky than mavenCentral()
+            url "https://maven-central.storage-download.googleapis.com/repos/central/data/"
+        }
```

Comment:
@zpencer can this google mirror apply to android build.gradle files?

Reply:
We can try it out and see if it works. I'm not sure if there's anything special about android related packages.